spec/controllers/oauth_controller_spec.rb

```diff
@@ -31,7 +31,7 @@
   end
 
   describe 'create_omniauth' do
-    let(:user) { Maestrano::Connector::Rails::User.new(email: 'lla@mail.com', tenant: 'default') }
+    let(:user)       { Maestrano::Connector::Rails::User.new(email: 'lla@mail.com', tenant: 'default') }
```

Comment:
I put some longer variables and then removed them, forgot the tab! Will remove it